charts/longhorn/100.2.1+up1.3.0-rc3/Chart.yaml

```diff
@@ -0,0 +1,41 @@
+annotations:
+  catalog.cattle.io/auto-install: longhorn-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Longhorn
+  catalog.cattle.io/kube-version: '>=1.18.0-0'
```

Comment:
Please update this upstream to the format `'>= 1.18.0-0 < 1.XX.0-0'` where XX is the max Kubernetes version the chart support +1. More on the reasoning here: https://github.com/rancher/charts/pull/1929

Reply:
Resolved. cc @PhanLe1010